torch/quantization/fx/quantize.py

```diff
@@ -199,7 +206,7 @@ def get_qconfig(module):
             elif node.op == 'call_module':
                 self.qconfig_map[node.name] = get_qconfig(self.modules[node.target])
 
-    def _prepare(self, model, qconfig_dict, inplace, is_dynamic_quant):
+    def _prepare(self, model, qconfig_dict, inplace, is_dynamic_quant, is_child_module):
```

Comment:
can we document this extra argument here?  It's hard to understand for someone who hasn't seen this PR recently.

Reply:
this is an internal API, docs are in prepare_child_module_fx

Reply:
yes, this would help with readability for people who work on this code (not for users of quantization)

Reply:
I don't want to repeat the docs though, would it help if I use `see prepare_child_module_fx for docs` for is_child_module?